src/Avalonia.Controls/Notifications/Notification.cs

```diff
@@ -0,0 +1,60 @@
+﻿// Copyright (c) The Avalonia Project. All rights reserved.
+// Licensed under the MIT license. See licence.md file in the project root for full license information.
+
+using System;
+
+namespace Avalonia.Controls.Notifications
+{
+    /// <summary>
+    /// Implements the INotification interfaces.
+    /// Can be displayed by both <see cref="INotificationManager"/> and <see cref="IManagedNotificationManager"/>
+    /// </summary>
+    /// <remarks>
+    /// This notification content type is compatible with native notifications.
+    /// </remarks>
+    public class Notification : INotification
+    {
+        /// <summary>
+        /// Instantiates an instance of <see cref="Notification"/>
+        /// </summary>
+        /// <param name="title">The title of the notification.</param>
+        /// <param name="message">The message to be displayed in the notification.</param>
+        /// <param name="type">The <see cref="NotificationType"/> of the notification.</param>
+        /// <param name="expiration">The expiry time at which the notification will close. 
+        /// Use <see cref="TimeSpan.Zero"/> for notifications that will remain open.</param>
+        /// <param name="onClick">The Action to call when the notification is clicked.</param>
```

Comment:
Maybe use "an" here instead of "the" because "the" to me suggests that it's mandatory. Or is that just me? ;)

```suggestion
        /// <param name="onClick">An Action to call when the notification is clicked.</param>
```